src/components/v2/common/message.ui.tsx

```diff
@@ -52,20 +58,25 @@ const MessageUI: React.FC<MsgUIProps> = ({
                                 />
                             );
                         case MsgUIType.MULTI_CONTAINER:
-                            return <MultipleContainer />;
+                            return <MultipleContainer className={iconClassName || ''} />;
                         default:
-                            return <InfoIcon className="fcn-0" width={size} height={size} />;
+                            return <InfoIcon className={`fcn-0 ${iconClassName || ''}`} width={size} height={size} />;
                     }
                 })()}
             </div>
-            <div className="fs-14" style={{ ...msgStyle, marginTop: '8px', color: 'white' }}>
+            <div className="fs-14" style={{ marginTop: '8px', color: 'white', ...msgStyle }}>
                 {msg}
             </div>
             {isShowActionButton && (
                 <div
                     className="cursor"
                     onClick={onActionButtonClick}
-                    style={{ fontSize: '14px', textDecoration: 'underline', color: 'var(--B300)' }}
+                    style={{
+                        fontSize: '14px',
+                        textDecoration: 'underline',
+                        color: 'var(--B300)',
+                        ...actionButtonStyle,
```

Comment:
if actionButtonStyle is not provided (i.e. null), please test that part also. (i.e. regression)

Reply:
Yes. It's already been tested and it works perfectly fine without any errors.